contracts.md

```diff
@@ -56,16 +57,26 @@ are for instance the [BCV](https://docs.olympusdao.finance/references/glossary#b
 and the max individual payout. Below are listed old DAI bond contracts which got
 phased out due to historical reasons.
 
-- https://etherscan.io/address/0xa64ED1b66Cb2838Ef2A198D8345c0ce6967A2A3c
+- V1 https://etherscan.io/address/0xa64ED1b66Cb2838Ef2A198D8345c0ce6967A2A3c
+- V2 https://etherscan.io/address/0xd03056323b7a63e2095ae97fa1ad92e4820ff045
+- V3 https://etherscan.io/address/0x575409F8d77c12B05feD8B455815f0e54797381c
```

Comment:
Can you remove the old contract address at line 51?

Reply:
Done